console/src/views/HVAC.js

```diff
@@ -75,7 +123,59 @@ const HvacBase = kind({
 	)
 });
 
-const Hvac = SaveLayoutArrangement('hvac')(HvacBase);
+const defaultConfig = {
+	acSelected: false,
+	autoSelected: false,
+	speeds: ['Off', 'Low', 'Medium', 'High'],
+	temps: ['HI', '74°', '73°', '72°', '71°', '70°', '69°', '68°', '67°', '66°', 'LO']
+};
+
+const HvacDecorator = hoc(defaultConfig, (configHoc, Wrapped) => {
+	return class extends React.Component {
+		static displayName = HvacDecorator;
+
+		constructor (props) {
+			super(props);
+
+			this.state = {
+				acSelected: configHoc.acSelected,
```

Comment:
Could these (`acSelected` and `autoSelected`) be set initially via props, with the callbacks and values coming from the  AppStateConnect component? This way, the AC settings could be saved in the current user profile and change automatically when the user is switched.

There's a decent example of how to use it in the new USerSelectionPopup code, down at the bottom.
Ex: 
```
const UserSelectionPopup = AppContextConnect(({userId, updateAppState}) => ({
	userId: userId,
	updateUser: ({selected}) => {
		updateAppState((state) => {
			state.userId = selected + 1;
		});
	}
}))(UserSelectionPopupBase);
```
In this code example, `AppContextConnect` is providing the keys from the defaults (and current/live values) assigned in the `AppContextProvider`, and making them available as "props" that we can send to the wrapped component as real props (userId). We also add an update method which takes the `selected` key from the `updateUser` callback (being provided as a new prop to the wrapped component) and assign it to the app state which comes in from `updateAppState`.

There are other examples of how to save into the user's profile by destructuring `userSettings` in that first line and saving back to it below in places like DisplaySettings.js, but this example is the most straightforward I was able to quickly locate.